assets/js/event-handler.js

```diff
@@ -30,17 +31,19 @@ document.getElementById('bug-report').addEventListener('vl-bug-report', (e) => {
       background: "#a5dc86",
       title: 'Bug Reported Successfully',
     })
-  } else {
+  } else {    
+    const error = event.detail.error;    
     Toast.fire({
       icon: 'error',
       iconColor: "white",
       color: "white",
       background: "#f27474",
       timer: 5000,
       title: 'Bug Report Failed, Please Try Again',
```

Comment:
It's be good to shorten the title and move some of the details into the text section